src/tests/api/team/api.test.ts

```diff
@@ -21,8 +22,34 @@ describe('[TEAMS API]', () => {
 		token = signInUser(user)
 	})
 
+	describe('GET /api/teams', () => {
+		it(`[${roleWithReadTeam}]. should return 400 user id is not provided as a param`, async () => {
+			// Action
+			const result = await apiRequest
+				.get('/api/teams')
+				.set('Authorization', token)
+			// Expect
+			expect(result.status).toEqual(httpStatus.BAD_REQUEST)
+		})
+	})
+
+	describe('GET /api/teams', () => {
+		it(`[${roleWithReadTeam}]. should return 200 with teams that user belongs to`, async () => {
+			// Arrange
+			await addTeam(createMockTeam(user._id))
+			// Action
+			const result = await apiRequest
+				.get('/api/teams')
+				.set('Authorization', token)
+				.query({userId: user._id.toString()})
+
+			// Expect
+			expect(result.status).toEqual(httpStatus.OK)
```

Comment:
should bootstrap user's teams. Add one or two teams to db and assign the ids to user object